code/drasil-printers/lib/Language/Drasil/Printing/Import/Symbol.hs

```diff
@@ -17,13 +17,18 @@ symbol (Integ    n) = P.Int (toInteger n)
 symbol (Special  s) = P.Spec s
 --symbol (Greek g)    = P.Gr g
 symbol (Concat  sl) = P.Row $ map symbol sl
---
--- Handle the special cases first, then general case
-symbol (Corners [] [] [x] [] s) = P.Row [P.Row [symbol s, P.Sup $ symbol x]]
-symbol (Corners [] [] [] [x] s) = P.Row [P.Row [symbol s, P.Sub $ symbol x]]
-symbol (Corners [_] [] [] [] _) = error "rendering of ul prescript"
-symbol (Corners [] [_] [] [] _) = error "rendering of ll prescript"
-symbol Corners{}                = error "rendering of Corners (general)"
+symbol (Corners ul dl ur dr s)  =
+  let
+    mergeSymbols :: [Symbol] -> P.Expr
+    mergeSymbols [s'] = symbol s'
+    mergeSymbols ss   = P.Row $ map symbol ss 
+
+    renderSubSup :: [Symbol] -> [Symbol] -> P.Expr -> [P.Expr]
+    renderSubSup [] [] _ = []
+    renderSubSup us [] x = [x, P.Sup $ mergeSymbols us]
+    renderSubSup [] ds x = [x, P.Sub $ mergeSymbols ds]
+    renderSubSup us ds x = [x, P.Sup $ mergeSymbols us, P.Sub $ mergeSymbols ds]
```

Comment:
I'm guessing this is the line than switches the order of Sup and Sub "by mistake".

Reply:
Yep! Do you prefer the other way?